packages/expo-barcode-scanner/android/src/main/java/expo/modules/barcodescanner/BarCodeScannedEvent.java

```diff
@@ -1,32 +1,45 @@
 package expo.modules.barcodescanner;
 
+import android.graphics.Point;
 import android.os.Bundle;
+import android.os.Parcelable;
 import android.support.v4.util.Pools;
+import android.util.Pair;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 
 import expo.core.interfaces.services.EventEmitter;
 import expo.interfaces.barcodescanner.BarCodeScannerResult;
+import expo.modules.barcodescanner.utils.Helper;
 
 public class BarCodeScannedEvent extends EventEmitter.BaseEvent {
   private static final Pools.SynchronizedPool<BarCodeScannedEvent> EVENTS_POOL =
       new Pools.SynchronizedPool<>(3);
 
   private BarCodeScannerResult mBarCode;
   private int mViewTag;
+  private List<Bundle> mCornerPoints;
+  private Bundle mBoundingBox;
 
   private BarCodeScannedEvent() {}
 
-  public static BarCodeScannedEvent obtain(int viewTag, BarCodeScannerResult barCode) {
+  public static BarCodeScannedEvent obtain(int viewTag, BarCodeScannerResult barCode, float density) {
```

Comment:
Let's leave `BarCodeScannedEvent` to still only receive `BarCodeScannerResult`, as it will have points already converted and will export methods to get bounds and corner points